meta-facebook/yv35-cl/src/main.c

```diff
@@ -42,8 +48,7 @@ void set_sys_status()
 
 void main(void)
 {
-	uint8_t proj_stage = (FIRMWARE_REVISION_1 & 0xf0) >> 4;
-	printk("Hello, wellcome to yv35 craterlake %x%x.%x.%x\n", BIC_FW_YEAR_MSB, BIC_FW_YEAR_LSB,
+	printf("Hello, wellcome to yv35 craterlake %x%x.%x.%x\n", BIC_FW_YEAR_MSB, BIC_FW_YEAR_LSB,
```

Comment:
welcome

Reply:
We have fixed this and done the forced update.
Please review it. Thanks.